packages/ui-navigation/src/Navigation/README.md

```diff
@@ -20,7 +20,7 @@ example: true
         icon={<Img src={placeholderLogo(85, 85)} constrain="cover" />}
         label={<ScreenReaderContent>Home</ScreenReaderContent>}
         href="#"
-        theme={{
+        themeOverride={{
```

Comment:
good catch, we should aways check readmes and examples.

Reply:
It made me think: can we make this work with the old `theme={{}}` syntax? It will be quite annoying to replace all overrides for every component, pretty sure people use it quite often... Don't fix it in this component, just let's talk about it next time :)